scripts/maint/practracker/practracker.py

```diff
@@ -164,6 +165,8 @@ def main(argv):
     parser = argparse.ArgumentParser(prog=progname)
     parser.add_argument("--regen", action="store_true",
                         help="Regenerate the exceptions file")
+    parser.add_argument("--list-overstrict", action="store_true",
```

Comment:
I think this option should be called "list overbroad" or something.

Double negatives are hard, but I don't know how to eliminate this one. Here are some possible options:
* list excess(ive)
* list unnecessary
* list better (that is, we are doing better than our exceptions)

Reply:
19536fd18d02a5 fixes this one.